Packages/com.unity.inputsystem/CHANGELOG.md

```diff
@@ -34,6 +34,7 @@ however, it has to be formatted properly to pass verification tests.
 - Fixed issue where composite part dropdown manipulates binding path and leaves composite part field unchanged.
 - Fixed lingering highlight effect on Save Asset button after clicking.
 - Fixed missing name in window title for Input Action assets.
+- Fixed InputManager.asset file growing in size on each Reset call (ISX-1839)
```

Comment:
Minor: Missing full-stop in end. Referencing internal case ID for a non-publicly visible issue.

Reply:
Changed (online in githib editor ;) )
Note: Some prior change log entries also seem to have some internal bug references but I've not changed those in this PR